backend/services/core/src/datasources/GoverningDocumentsAPI.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  UUID,
+  context,
+  dbUtils,
+} from '../shared';
+
+import * as gql from '../types/graphql';
+import * as sql from '../types/governingDocuments';
+
+const TABLE_NAME = 'governing_documents';
+
+export function convertGoverningDocument(document: sql.GoverningDocument): gql.GoverningDocument {
+  return {
+    id: document.id,
+    title: document.title,
+    url: document.url,
+    type: document.document_type,
```

Comment:
Perhaps spread and just overwrite type?

Reply:
I prefer being expressive in this context to be more clear.